app.go

```diff
@@ -648,7 +650,7 @@ func VisualizeError(err error) (string, error) {
 // Start, Done, and Stop methods. Applications with more specialized needs
 // can use those methods directly instead of relying on Run.
 func (app *App) Run() {
-	app.run(app.Done())
+	os.Exit(app.run(app.Done()))
```

Comment:
Looks like when the exit code is 0, we're changing the behavior? i.e. we didn't `os.Exit` before with exit code 0 ever, but it would now. If someone relied on call to this returning to the caller, wouldn't that break them?

I suppose the docs do say apps with more specialized needs should use `Start`, `Done` and `Stop` methods but now that we're changing that behavior, would this be considered a breaking change?

Reply:
Ah, good catch. Typically, people who use app.Run are ceding control of the application, but I'd like to not change the behavior here in case someone is relying on this returning at some point. Fixing.